Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright 2015-2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Management.Automation;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Creates a new empty dataset in the specified project.
+    /// </para>
+    /// <para type="description">
+    /// Creates a new empty dataset in the specified project.  This command can either take a premade Dataset object, 
+    /// or build one using parameters.  Premade Datasets can be passed via the pipeline or with the -Dataset parameter.
+    /// If both are supplied, the premade dataset object will be used and the builder values will be ignored.  If no 
```

Comment:
Since you are using parameter set, powershell will raise an error if both parameter sets are used.